bootstrap/092_service_account_setup.sql

```diff
@@ -16,13 +16,14 @@ begin
         AS
         CALL ADMIN.UPGRADE_CHECK();
     grant MONITOR, OPERATE on TASK TASKS.UPGRADE_CHECK to APPLICATION ROLE ADMIN;
+    alter task TASKS.UPGRADE_CHECK resume;
 
     call internal.set_config('tenant_url', :web_url);
     call internal.set_config('url', :url);
 
     -- Bind the given reference ID to the 'OPSCENTER_API_INTEGRATION' reference. Must match the reference in manifest.yml
     -- differs from v1 in that all external functions are not re-created by update_reference()
-    call admin.update_reference('OPSCENTER_API_INTEGRATION', 'finalize_setup_from_service_account', :api_integration_ref_id);
+    call admin.update_reference('OPSCENTER_API_INTEGRATION', 'ADD', :api_integration_ref_id);
```

Comment:
I thikn the update ref command was always broken?

Reply:
system$set_reference has to be called by a user and not the native app, IIRC. We should replace this with an insert to `internal.reference_management` or add some new "operation".

Reply:
ahhhh...ok. I will just add the insert here

Reply:
Yeah, something that does an `insert` seems reasonable. Taking a step back, I'm not really sure if we had a goal with the reference_management table (or if we just copied the public examples), but I don't think we have anything that actually reads from it today.